src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -7,6 +7,8 @@
 
 import Container from 'components/Container';
 import Flex from 'components/Flex';
+// $FlowFixMe
+import MDXRenderer from 'gatsby-plugin-mdx/mdx-renderer';
```

Comment:
Was getting this error:

```
Error: src/components/MarkdownPage/MarkdownPage.js:10
 10: import MDXRenderer from 'gatsby-plugin-mdx/mdx-renderer';
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ gatsby-plugin-mdx/mdx-renderer. Required module not found
```

Not sure why though. So I added `$FlowFixMe` to get this PR to build. Anyone have any ideas why flow wasn't happy with this import?